kitchen-openstack.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler'
   spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop', '0.34.1'
```

Comment:
This is a decently old rubocop - I'd recommend using 0.35.1 (which is what [chefstyle](https://github.com/chef/chefstyle)) uses, or setting it to `~> 0.36` to pick up the latest in the 0.x series.